packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope.ts

```diff
@@ -236,7 +239,7 @@ function createSoapBody({
   const toUrl = gateway.url;
   const providerId = bodyData.principalCareProviderIds[0];
   const homeCommunityId = getHomeCommunityId(gateway, bodyData.samlAttributes);
-  const patientGender = bodyData.patientResource.gender === "female" ? "F" : "M";
+  const patientGender = mapFhirToGenderAtBirth(bodyData.patientResource.gender);
```

Comment:
Shouldn't we have a mapping specific to CQ? I don't think we should tie our "gender at birth" to it.

Reply:
this relates to the other discussion about having different types for the IHE gateway gender and our Metriport Gender. Right now they are the same so imo its appropriate to use our GenderAtBirth. This should change pending our decision on the other thread